Surface_mesh_topology/doc/Surface_mesh_topology/dependencies

```diff
@@ -0,0 +1,6 @@
+Combinatorial_map
+Generalized_map
+Linear_cell_complex
+BGL
+Surface_mesh
+Polyhedron
```

Comment:
@gdamiand Add `Manual` in that list, because `licensesGPL` is defined in `Documentation/doc/Documentation/License.txt` inside the `Manual` package.

Reply:
Thanks @lrineau  !!